### ISSUE TITLE
SOCIAL_FRIENDS_USING_ALLAUTH does not default to False

### ISSUE THREAD

**Issue by ghost** [laplacesdemon/django-social-friends-finder#7]
The docs say "SOCIAL_FRIENDS_USING_ALLAUTH: True if you use `django-allauth`. Default is False". So I did not set it since the default is ok for me, I want it to be False (I don't Use allauth, but django-social-auth so I understand it should be false).
Problem is, when I try to load the friends page I receive an error that no module 'allauth.socialaccount.models' exists. That's because in the module backends/facebook_backend.py the line 3 checks if there is a 'False' corresponding to that setting. It does so using the function in utils.setting which in turn checks in the settings. If it finds nothing the it defaults to 'default' which in this case is 'None'. So actually with that logic it sort of defaults to True. To correct it, I don't know if the default=None is here for something else too or if it could be changed to False. If that's not possible then I guess updating the logic in the facebook backend check could be easy to implement. In the meantime I will set that setting to false in my project but it would be nice to actually default to False. If you tell me which solution fits best I can do it and submit a pull request.

UPDATE: I was very wrong about my understanding of the logic in the facebook backend it actually gives a value to default which is indeed False.

**Comment by ghost** [comment]
Ok I know where my confusion comes from. I am testing the package with the version that pip gives. And I am reading the code here on github in the master branch. And they differ. In the pypi version the facebook backend on line 3 reads:
if setting("SOCIAL_FRIENDS_USING_ALLAUTH", False):
and the same file on github (master) reads:
if not setting("SOCIAL_FRIENDS_USING_ALLAUTH", False):

So I thought this must be the problem and that's when I opened the downloaded package that I realized it is not up to date.
The latest commit on that file reads 'Oops!' May be it refers to that, or not. But it is 6 months old so if pypi has not been updated in that much time it's strange that nobody realized that before.

**Comment by laplacesdemon** [comment]
The pypi release must be 0.2 which should be the latest version if I'm not mistaken. I will make sure it is, if not I will create a new release on pypi.

Are you sure that you're using the correct version on pypi

**Comment by laplacesdemon** [comment]
indeed, you're right. the pypi package is out dated. I will update it as soon as I can. 

In the mean time you can use directly from this repo.

**Comment by ghost** [comment]
Thanks, I already corrected my local version. Just wanted to check if the pypi package was right. Thanks for your answer.
